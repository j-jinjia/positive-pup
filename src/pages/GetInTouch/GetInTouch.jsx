import ContactHeader from "../../components/ContactHeader/ContactHeader";
import Layout from "../../components/Layout/Layout";

const GetInTouch = () => {
  return (
    <Layout>
      <ContactHeader
        headingText={"Contact Us"}
        subheadingText={"I'd love to hear from you!"}
        isLeftAlign={true}
      />
    </Layout>
  );
};

export default GetInTouch;
