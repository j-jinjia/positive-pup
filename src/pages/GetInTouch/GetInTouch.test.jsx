import { customRender } from "../../utils/testUtils";
import GetInTouch from "./GetInTouch";

it("should render the footer component", () => {
  const { container } = customRender(<GetInTouch />);

  expect(container).toMatchSnapshot();
});