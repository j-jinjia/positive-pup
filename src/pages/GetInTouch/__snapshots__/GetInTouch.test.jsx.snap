// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the GetInTouch page 1`] = `
<div>
  <div
    class="layout"
  >
    <div
<<<<<<< HEAD
      class="navbar"
    >
      <div
        class="navbar__container"
      >
        <a
          href="/"
        >
          <img
            alt="The Positive Pup logo"
            class="navbar__container-logo"
            src="footer-logo.svg"
          />
        </a>
        <button
          class="navbar__container-button"
        >
          <img
            alt="Open site menu"
            class="navbar__container-button-image"
            src="hamburger-menu.svg"
          />
        </button>
      </div>
    </div>
    <h1>
      Get in touch
    </h1>
=======
      class="contact-header"
    >
      <h1
        class="contact-header__title"
      >
        CONTACT US
      </h1>
      <h2
        class="contact-header__heading"
      >
        I'd love to hear from you!
      </h2>
      <p
        class="contact-header__text"
      >
        Please fill in the form or alternatively, drop me an email using the link below.
      </p>
      <a
        class="contact-header__anchor"
        href="mailto:info@thepositivepup.co.uk"
      >
        <img
          alt=""
          src="mail.svg"
        />
        <p
          class="contact-header__button-text"
        >
          info@thepositivepup.co.uk
        </p>
      </a>
    </div>
>>>>>>> d0856e6ee34185512949d7ad7ee76ba26ed0604c
    <div
      class="footer"
    >
      <div
        class="footer__image-container"
      >
        <img
          alt="logo"
          class="footer__logo"
          src="footer-logo.svg"
        />
      </div>
      <ul
        class="footer__links-container"
      >
        <li
          class="footer__list-item"
        >
          <a
            class="footer__link"
            href="/about"
          >
            About
          </a>
        </li>
        <li
          class="footer__list-item"
        >
          <a
            class="footer__link"
            href="/services"
          >
            Services
          </a>
        </li>
        <li
          class="footer__list-item"
        >
          <a
            class="footer__link"
            href="/gallery"
          >
            Gallery
          </a>
        </li>
        <li
<<<<<<< HEAD
          class="footer__list-item"
=======
          class="footer__list-item footer__list-item--desktop"
        >
          <a
            class="footer__link"
            href="/terms-and-conditions"
          >
            Terms & Conditions
          </a>
        </li>
        <li
          class="footer__list-item footer__list-item--mobile"
>>>>>>> d0856e6ee34185512949d7ad7ee76ba26ed0604c
        >
          <a
            class="footer__link"
            href="/terms-and-conditions"
          >
<<<<<<< HEAD
            Terms and conditions
=======
            T&C's
>>>>>>> d0856e6ee34185512949d7ad7ee76ba26ed0604c
          </a>
        </li>
        <li
          class="footer__list-item"
        >
          <a
            class="footer__link"
            href="/faqs"
          >
<<<<<<< HEAD
            FAQs
=======
            FAQ's
>>>>>>> d0856e6ee34185512949d7ad7ee76ba26ed0604c
          </a>
        </li>
        <li
          class="footer__list-item"
        >
          <a
            class="footer__link"
            href="/get-in-touch"
          >
            Contact
          </a>
        </li>
      </ul>
      <hr
        class="footer__line-break"
      />
      <div
        class="footer__secondary-links-container"
      >
        <div
          class="footer__copyright-text"
        >
          Â© 2022 The Positive Pup. All rights reserved.
        </div>
        <div
          class="footer__social-links"
        >
          <a
            href="https://www.facebook.com/thepositivepupdogtraining"
            target="blank"
          >
            <img
              alt="link to facebook"
              src="facebook-logo.svg"
            />
          </a>
          <a
            href="https://www.instagram.com/the_positivepup/"
            target="blank"
          >
            <img
              alt="link to instagram"
              src="instagram-logo.svg"
            />
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;
